src/ocf_io.c

```diff
@@ -54,6 +54,9 @@ struct ocf_io *ocf_io_new(ocf_volume_t volume)
 	struct ocf_io_meta *io_meta;
 	void *data;
 
+	if (!ocf_refcnt_inc(&volume->refcnt))
+		return NULL;
+
 	data = env_allocator_new(volume->type->allocator);
 	if (!data)
 		return NULL;
```

Comment:
We need to decrease refcnt in error path

Reply:
done